Source/levels/gendung.h

```diff
@@ -213,7 +215,10 @@ extern int16_t dMonster[MAXDUNX][MAXDUNY];
  * dDead[x][y] >> 0x5 - direction
  */
 extern DVL_API_FOR_TEST int8_t dCorpse[MAXDUNX][MAXDUNY];
-/** Contains the object numbers (objects array indices) of the map. */
+/** 
+* Contains the object numbers (objects array indices) of the map.
+* Large objects have negative id for its extended area
+*/

```

Comment:
```suggestion
 * Contains the object numbers (objects array indices) of the map.
 * Large objects have negative id for there extended area
 */

```

Reply:
Ack, I was too late editing AJenbo's suggestion. It should be "their", not "there".